### PR TITLE
ARROW-12029: [R] Remove args from FeatherReader$create v2

### PR DESCRIPTION
**Pull request by pachadotdev**

### CODE REVIEW

**Comment by jonkeane** [comment]
I don't think that this will cause a conflict since you have the same change, but it looks like this branch has not been rebased on `apache/arrow@master`. I see this because https://github.com/apache/arrow/pull/9757/files#diff-3cda3c18dfe3014eabcb3854645f438184492bd1e9e4fc153439f01a609e74d0L209 has a change, but that change has already been merged. This might be a good time to practice rebasing since there aren't a large number of changes in this branch. Would you mind rebasing against the most recent commits in apache/arrow?

**Comment by pachamaltese** [comment]
> I don't think that this will cause a conflict since you have the same change, but it looks like this branch has not been rebased on `apache/arrow@master`. I see this because https://github.com/apache/arrow/pull/9757/files#diff-3cda3c18dfe3014eabcb3854645f438184492bd1e9e4fc153439f01a609e74d0L209 has a change, but that change has already been merged. This might be a good time to practice rebasing since there aren't a large number of changes in this branch. Would you mind rebasing against the most recent commits in apache/arrow?

thanks, I shall create a new PR since I added many files I didn't want to

File: r/R/feather.R [hunk 1]
```diff
@@ -178,8 +178,6 @@ read_feather <- function(file, col_select = NULL, as_data_frame = TRUE, ...) {
 #' takes the following arguments:
```
Comment (nealrichardson) [inline r/R/feather.R:178]:
```suggestion
#' takes the following argument:
```

**Review by nealrichardson — CHANGES_REQUESTED** [review]
Please also regenerate the Rd docs

**Review by nealrichardson — APPROVED** [review]

**Pull request closed without merging**